spec/unit/screen_spec.rb

```diff
@@ -271,6 +271,15 @@ def ioctl(_control, buf)
       end
     end
 
+    it "doesn't detect size if the output doesn't respond to #iotcl" do
```

Comment:
The test description isn't strictly correct. At the module level, there is a check to see whether output supports `ioctl`, failing that, it inlines the `size_from_ioctl` method body with `nil` implementation. The issue here is the runtime swapping of the output stream.

```suggestion
    it "doesn't detect size if the output changes to non-ioctl at runtime" do
```